front/src/utils/date.ts

```diff
@@ -11,8 +12,16 @@ export function formatIsoDate(date: Date) {
   return date.toISOString().substring(0, 10);
 }
 
-export function dateTimeFrenchFormatting(date: Date) {
-  return dayjs.utc(date).tz(dayjs.tz.guess()).format('D MMM YYYY HH:mm').replace(/\./gi, '');
+export function dateTimeFormatting(date: Date, withoutTime: boolean = false) {
+  switch (i18n.language) {
+    case 'fr':
+      dayjs.locale('fr');
+      break;
+    default:
+      dayjs.locale('en-gb');
+  }
+  const dateFormat = withoutTime ? 'D MMM YYYY' : 'D MMM YYYY HH:mm';
```

Comment:
The date in American English is written as "MMM D YYYY", should we consider to use this instead of the British date ?